### PR TITLE
Update Authors

### PR DESCRIPTION
**Pull request by aktech**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -464,4 +464,9 @@ Tanu Hari Dixit <tokencolour@gmail.com>
 Shekhar Prasad Rajak <shekharrajak@live.com>
 Aqnouch Mohammed <aqnouch.mohammed@gmail.com>
 Arafat Dad Khan <arafat.da.khan@gmail.com>
+Boris Atamanovskiy <shaomoron@gmail.com>
+Arafatk <arafat@iitkpg.ac.in>
 Sam Tygier <sam.tygier@hep.manchester.ac.uk>
+alkis akritas <akritas@uth.gr>
+Jai Luthra <me@jailuthra.in>
+MSeeker <Seeker1995@gmail.com>
```
Comment (asmeurer) [inline AUTHORS]: Also update .mailmap so that these are full names (with proper capitalization) where possible.

**Comment by asmeurer** [comment]
The authors updating script should print some message about updating .mailmap, which can be checked with the mailmap update script but some degree of it has to be done manually.

**Comment by aktech** [comment]
ping @leosartaj

**Comment by asmeurer** [comment]
Please normalize the author names, and make sure none are duplicated.

**Comment by aktech** [comment]
> Please normalize the author names, and make sure none are duplicated.

Checked for duplicates, and normalised as well.

**Comment by asmeurer** [comment]
Some of the names need spaces. 

Also, you should update the .mailmap file (there's a script `bin/mailmap_update.py` to help).

File: AUTHORS [hunk 2]
```diff
@@ -512,3 +513,17 @@ Nathan Musoke <nathan.musoke@gmail.com>
 Abhishek Garg <abhishekgarg119@gmail.com>
 Dana Jacobsen <dana@acm.org>
 Vasiliy Dommes <vasdommes@gmail.com>
+Phillip Berndt <phillip.berndt@googlemail.com>
+Haimo Zhang <zh.hammer.dev@gmail.com>
+Subham Tibra <shubh.tibra@gmail.com>
```
Comment (sampadsaha5) [inline AUTHORS]: Duplicate at L473.

Comment (asmeurer) [inline AUTHORS]: I thought there was an "h" in his name. Good to know I wasn't just misremembering that. @shubhamtibra which spelling do you prefer?

Comment (shubhamtibra) [inline AUTHORS]: `Shubham Tibra` is fine. It's already there.

Comment (aktech) [inline AUTHORS]: Was checking duplicate manually, didn't saw it due to different spelling.

Comment (shubhamtibra) [inline AUTHORS]: Cool :)

**Pull request closed without merging**